io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl

```diff
@@ -0,0 +1,164 @@
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:lifecycleContext a samm:Property ;
+   samm:preferredName "Lifecycle Context"@en ;
+   samm:description "The lifecycle context in which the child part was assembled into the parent part."@en ;
+   samm:characteristic :LifecycleContextCharacteristic ;
+   samm:exampleValue "AsBuilt" .
+
+:businessPartner a samm:Property ;
+   samm:preferredName "Business Partner"@en ;
+   samm:description "The supplier of the given child item."@en ;
+   samm:characteristic :BpnTrait ;
+   samm:exampleValue "BPNL50096894aNXY" .
+
+:QuantityCharacteristic a samm-c:Quantifiable ;
+   samm:description "Describes the quantity in which the child part is assembled in the given parent object by providing a quantity value and the measurement unit in which the quantity is measured."@en ;
+   samm:dataType :Quantity .
+
+:LifecycleContextCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Lifecycle Context"@en ;
+   samm:description "The possible values for lifecycle context as defined by STEP: AsRequired, AsDesigned, AsPlanned, AsBuilt, AsMaintained, AsRecycled"@en ;
+   samm:see <http://private.pdm-if.org/web/pdm-if/recommended-practices1> ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "AsRequired" "AsDesigned" "AsPlanned" "AsBuilt" "AsMaintained" "AsRecycled" ) .
+
+:BpnTrait a samm-c:Trait ;
+   samm:preferredName "BPN Business Partner Number"@en ;
+   samm:description "BPN Business Partner Number"@en ;
+   samm-c:baseCharacteristic :BpnCharacteristic ;
+   samm-c:constraint :BpnConstraint .
+
+:Quantity a samm:Entity ;
+   samm:description "Comprises the number of objects and the unit of measurement for the respective child objects"@en ;
+   samm:properties ( :quantityNumber :measurementUnit ) .
+
+:BpnCharacteristic a samm:Characteristic ;
+   samm:preferredName "BPN Characteristic"@en ;
+   samm:description "BPN Characteristic"@en ;
+   samm:dataType xsd:string .
+
+:BpnConstraint a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "BPN Constraint"@en ;
+   samm:description "Business Partner Number Contraint only BPNL wich stands for a legal entity is allowed."@en ;
+   samm:value "^(BPNL)(\\d{8})([a-zA-Z0-9]{4})$" .
+
+:quantityNumber a samm:Property ;
+   samm:preferredName "Quantity Number"@en ;
+   samm:description "The number of objects related to the measurement unit"@en ;
+   samm:characteristic :NumberofObjects ;
+   samm:exampleValue "2.5"^^xsd:double .
+
+:measurementUnit a samm:Property ;
+   samm:description "Unit of Measurement for the quantity of serialized objects"@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/2.0.0/appendix/unitcatalog.html> ;
+   samm:characteristic samm-c:UnitReference ;
+   samm:exampleValue "unit:litre"^^samm:curie .
+
+:NumberofObjects a samm:Characteristic ;
+   samm:description "Quantifiable number of objects in reference to the measurementUnit"@en ;
+   samm:dataType xsd:double .
+
```

Comment:
Release notes are missing (just copy from another model and adjust accordingly). Please also indicate where the model originates from